cache/bug_excerpt.go

```diff
@@ -17,8 +18,10 @@ type BugExcerpt struct {
 	CreateUnixTime    int64
 	EditUnixTime      int64
 
+	Title  string
 	Status bug.Status
 	Author bug.Person
+	NoOfComments int
```

Comment:
It's not wrong, but let's try to stay closer than golang vocabulary and name that one `LenComments`